test/components/views/emojipicker/EmojiPicker-test.tsx

```diff
@@ -24,6 +24,21 @@ import { stubClient } from "../../../test-utils";
 describe("EmojiPicker", function () {
     stubClient();
 
+    it("should not mangle default order after filtering", () => {
+        const ref = createRef<EmojiPicker>();
+        const { container } = render(
+            <EmojiPicker ref={ref} onChoose={(str: string) => false} onFinished={jest.fn()} />,
+        );
+
+        const beforeHtml = container.innerHTML;
+        //@ts-ignore private access
+        ref.current!.onChangeFilter("test");
+        expect(beforeHtml).not.toEqual(container.innerHTML);
+        //@ts-ignore private access
+        ref.current!.onChangeFilter("");
+        expect(beforeHtml).toEqual(container.innerHTML);
```

Comment:
some comments here wouldn't hurt

```suggestion
        // record the HTML before filtering
        const beforeHtml = container.innerHTML;
        
        // apply a filter and check the HTML has changed
        //@ts-ignore private access
        ref.current!.onChangeFilter("test");
        expect(beforeHtml).not.toEqual(container.innerHTML);
        
        // clear the filter and check that the HTML is as it was before
        //@ts-ignore private access
        ref.current!.onChangeFilter("");
        expect(beforeHtml).toEqual(container.innerHTML);
```